src/SBInclinedExponential.cpp

```diff
@@ -220,15 +220,11 @@ namespace galsim {
 
     double SBInclinedExponential::SBInclinedExponentialImpl::maxSB() const
     {
-        // When the disk is face on, the max SB is flux / 2 pi r0^2
-        // When the disk is edge on, the max SB is flux / 2 pi r0^2 * (r0/h0)
-        double maxsb = _flux * _inv_r0 * _inv_r0 / (2. * M_PI);
-        // The relationship for inclinations in between these is not linear.
-        // Empirically, it is vaguely linearish in sqrt(cosi), so we use that for
-        // the interpolation.  It's accurate to ~10-20% for moderate values of h0/r0.
-        double sc = sqrt(std::abs(_cosi));
-        maxsb *= (_h0 * sc + _r0 * (1.-sc)) / _h0;
-        return std::abs(maxsb);
+        // Maximum surface brightness isn't analytic, and is only needed for photon-shooting,
+        // so this is disabled.
+        throw std::runtime_error(
+            "The maxSB method of SBInclinedExponential is not yet implemented");
+        return 0;
```

Comment:
I got thinking about it again once I noticed that it was behaving even more poorly than expected for the new edge-on Sersic test case. It looks like the method is exposed to the user, so I didn't want them to falsely assume it would be accurate. I can put it back in for this and the InclinedSersic class though if you want (and perhaps have it output a warning?).